src/components/file-link.jsx

```diff
@@ -17,7 +17,7 @@ let MyBox = styled(Box)`
 export interface IFileLinkProps {
   icon: React.ComponentType<any>;
   href: string;
-  className?: string;
+  className?: ?string;
```

Comment:
I haven't heard about such types 🤔

Reply:
```javascript
type ClassType1 = { className?: string };
type ClassType2 = { className?: ?string };

const Component1 = (props: ClassType1) => (/* ... */);
const Component2 = (props: ClassType2) => (/* ... */);

<Component1 className={undefined} /> // error, because className - optional prop: string or not passed, but we have undefined
<Component2 className={undefined}  /> // ok, because className - optional prop: string or undefined/null
```